### PR TITLE
reuse App on displayForm

### PR DESCRIPTION
**Pull request by abbadon1334**
The old way to display an exception or a login form was to stop the current App and create a new one.
I manage to reuse the App, fix phpunit, remove deprecated code and do some cleanup.

### CODE REVIEW

File: demos/wizard.php [hunk 1]
```diff
@@ -4,25 +4,21 @@
...
 use atk4\ui\View;
-use atk4\ui\Wizard;
-use atk4\ui\App;
+use Throwable;
 
 include '../vendor/autoload.php';
+
 //include 'db.php';
 
 class wizard extends App
```
Comment (PhilippGrashoff) [inline demos/wizard.php]: Should be Uppercase

File: demos/wizard.php [hunk 2]
```diff
@@ -36,18 +32,19 @@ public function dbConnectFromWizard()
     }
 }
 
-$app = new AppWizard(); // App without authentication to be able to freely import data
+$app = new AppWizard(['title' => 'Agile Toolkit - Wizard setup']); // App without authentication to be able to freely import data
 $app->initLayout('Centered');
 
 /** @var Wizard $wizard */
-$wizard=$app->add('Wizard');
+$wizard = $app->add('Wizard');
```
Comment (PhilippGrashoff) [inline demos/wizard.php]: In the other files, you used ::addTo(). Would be great to have it here too to have demo code consistent

**Review by DarkSide666 — CHANGES_REQUESTED** [review]
Please address @PhilippGrashoff comments. Besides that looks good to me.

**Review by DarkSide666 — APPROVED** [review]
Now is good

**Pull request merged**